newspaper/article.py

```diff
@@ -520,17 +520,14 @@ def throw_if_not_downloaded_verbose(self):
         -> maybe throw ArticleException
         """
         if self.download_state == ArticleDownloadState.NOT_STARTED:
-            print('You must `download()` an article first!')
```

Comment:
The print statements instead of text embedded into a custom thrown exception was bad design from my part at the start. Thanks for this.

We should get in a habit of migrating all these print statements into custom exceptions or `LOG` messages at least.